--- gnotime-2.2.3/configure.in~	2007-09-24 16:42:19.000000000 +0200
+++ gnotime-2.2.3/configure.in	2007-12-23 00:59:40.628774264 +0100
@@ -192,7 +192,7 @@
 dnl *******************************
 
 GTKHTMLPRESENT=no
-for GTKHTMLVER in 3.8 3.7 3.6 3.5 3.4 3.2 3.1 3.0 ; do
+for GTKHTMLVER in 3.14 3.8 3.7 3.6 3.5 3.4 3.2 3.1 3.0 ; do
   if $PKG_CONFIG --exists libgtkhtml-$GTKHTMLVER ; then
 	PKG_CHECK_MODULES(LIBGTKHTML, libgtkhtml-$GTKHTMLVER >= $LIBGTKHTML_REQUIRED)
 	GTKHTMLPRESENT=yes
